### ISSUE TITLE
Add config support

### ISSUE THREAD

**Issue by srdecny** [srdecny/SubtitleComparer#7]
Implement a serializable config file for saving preferences.

**Comment by stranak** [comment]
Don't windows WPF apps have their own mechanism for user/app settings? I would probably go with the OS/framework standard approach, if possible. 

Possibly not the ideal approach, but mentions the key stuff, I think: http://geekswithblogs.net/mbcrump/archive/2010/06/17/configuring-applicationuser-settings-in-wpf-the-easy-way.aspx

**Comment by srdecny** [comment]
@stranak Yes, that feature was what I had in mind - it's just I'm not using it at all, and there are couple of places where I should use it - like setting up path to the directory with FFMpeg binaries.
